securing_apps/topics/oidc/nodejs-adapter.adoc

```diff
@@ -116,6 +116,24 @@ Once instantiated, install the middleware into your connect-capable app:
     app.use( keycloak.middleware() );
 ----
 
+==== Configuration for Proxies
+
+If the application is running behind a proxy that terminates an SSL connection
+express must be configured per the link:https://expressjs.com/en/guide/behind-proxies.html[express behind proxies] guide.
```

Comment:
I believe the word `express` should be `Express`

Thanks for submitting this Evan.

Reply:
Happy to help 👍 

Build is failing on a few links. Oddly it's reporting that this link in the PR returns a 403, but in the PR build it's working fine - https://keycloak-docs.github.io/deploy-docs-pullrequest/PR/728/server_development/index.html